GoogleSignIn/Sources/GIDSignInInternalOptions.m

```diff
@@ -25,6 +25,43 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @implementation GIDSignInInternalOptions
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
+                       presentingViewController:(nullable UIViewController *)presentingViewController
+                                      loginHint:(nullable NSString *)loginHint
+                                  addScopesFlow:(BOOL)addScopesFlow
+                               verifyCompletion:(nullable GIDVerifyCompletion)completion{
+  GIDSignInInternalOptions *options = [self defaultOptionsWithConfiguration:configuration
+                                                   presentingViewController:presentingViewController
+                                                                  loginHint:loginHint
+                                                              addScopesFlow:addScopesFlow
+                                                     accountDetailsToVerify:@[]
+                                                           verifyCompletion:completion];
+  return options;
+}
+
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
+                       presentingViewController:(nullable UIViewController *)presentingViewController
+                                      loginHint:(nullable NSString *)loginHint
+                                  addScopesFlow:(BOOL)addScopesFlow
+                         accountDetailsToVerify:(nullable NSArray *)accountDetailsToVerify
```

Comment:
Use lightweight generics with `NSArray`. Also shouldn't be nullable.

Reply:
Done.